kustomize/kustomize_generator_whitebox_test.go

```diff
@@ -49,6 +49,14 @@ func TestScanManifests(t *testing.T) {
 			base:    "./testdata/nokustomization/panic",
 			wantErr: true,
 		},
+		{
+			name: "utf-16LE encoding",
```

Comment:
Can we give this test a more descriptive name, please? What does it actually test? Something like "Files with UTF-16LE encoding should be included".

Reply:
Plus: This tests passes just fine without the changes to kustomize_generator.go.